ui-tests/src/test/java/org/openmrs/reference/page/EditPatientRelationshipPage.java

```diff
@@ -0,0 +1,36 @@
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.Select;
+
+public class EditPatientRelationshipPage  extends Page {
+
+    private  HomePage homePage;
+    private  static  final By SELECT_RELATIONSHIP_TYPE = By.id("relationship_type");
+    private static final By NAME = By.cssSelector("button, input");
+    private static final By NEXT_BUTTON = By.id("next-button");
+    private static final By CONFIRM_BUTTON = By.id("registration-submit");
+
+    public EditPatientRelationshipPage(RegistrationSummaryPage parent) {
+        super(parent);
+    }
+
+    public void clickOnSelectRelationshipType() throws InterruptedException{
+        clickOn(SELECT_RELATIONSHIP_TYPE);
+        // Create object of the select class
+        Select relationshipType = new Select(driver.findElement(By.id("relationship_type")));
+        // We need to loop through such that the driver can always use another relationship type
```

Comment:
Hello @kaweesi , I need  your help with in this line, i want to tell the driver to be able to loop through different relationshipstype not only restricting to only doctor attribute, inotherwords the driver can always look through different types of relationship, how can we implement this thanks

Reply:
you can pass a relationship type element into the method which you can pass into the feature file.

Reply:
With in the feature file , i passed doctor as relationship type ,  are you meaning something on this line
 relationshipType.selectByVisibleText("Doctor");